### PR TITLE
VXFM-5171 Inventory fails as host key checking seems to still occur on Ansible inventories.

### PR DESCRIPTION
**Pull request by aimers1975**
Provide method to remove host file entry which can be called before
ansible run.

### CODE REVIEW

File: lib/asm/util.rb [hunk 1]
```diff
@@ -37,6 +37,7 @@ module Util
     DEVICE_MODULE_PATH = "/etc/puppetlabs/puppet/modules"
     INSTALLER_OPTS_DIR = "/opt/razor-server/tasks/"
     DEVICE_LOG_PATH = "/opt/Dell/ASM/device"
+    HOST_FILE_PATH = "/home/razor/.ssh/known_hosts"
```
Comment (gavin-scott) [inline lib/asm/util.rb]: Grab this from the user's environment:

    File.join(ENV["HOME"], ".ssh", "known_hosts")

Comment (aimers1975) [reply lib/asm/util.rb]: Will do!

**Comment by aimers1975** [comment]
@gavin-scott Let me know if this is what you had in mind...or any other changes?

**Comment by aimers1975** [comment]
@gavin-scott Hopefully this is good to go!

**Pull request merged**